uefi-sct/SctPkg/TestCase/UEFI/EFI/BootServices/ProtocolHandlerServices/BlackBoxTest/ProtocolHandlerBBTestStress.c

```diff
@@ -151,6 +151,8 @@ EFI_GUID STRESS_TEST_NON_INTERFACE_PROTOCOL_38_GUID =
 EFI_GUID STRESS_TEST_NON_INTERFACE_PROTOCOL_39_GUID =
   { 0x169370a, 0xbc3d, 0x4326, {0x8a, 0x7, 0xa, 0x6c, 0x25, 0x73, 0x20, 0xd9 }};
 
+#define OPENINFO_LEVELS 3

```

Comment:
Took a further look into the code. 
The index is used for checking TempInfo which is a buffer returned by OpenProtocolInformation call. 
So I think a better name is OPENINFO_ENTRY_COUNT or OPENINFO_ENTRY_BUFFER_SIZE